### PR TITLE
add instructions for updating MariaDB manually (bsc#1132852)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

**Comment by rsalevsky** [comment]
This looks good but we should add the same section to the Ardana docs too.

File: xml/operations-maintenance-mariadb-manual-update.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="mariadb-manual-update">
+ <title>Updating &mariadb; with Galera</title>
+   <para>
+   When using Pacemaker, updating &mariadb; with Galera must be done
+   manually. &crow; will not install updates automatically. In particular, this
```
Comment (asettle) [inline xml/operations-maintenance-mariadb-manual-update.xml]: &crow; does not install updates...

File: xml/operations-maintenance-mariadb-manual-update.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   Update &mariadb; with the following procedure:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Using the Pacemaker GUI, put the cluster into maintenance mode. Detailed
+     information about the Pacemaker GUI and its operation is available in the
+     <link
+     xlink:href="https://www.suse.com/documentation/sle_ha/singlehtml/book_sleha/book_sleha.html#cha.ha.configuration.gui">&sle;
+     High Availability documentation</link>
```
Comment (asettle) [inline xml/operations-maintenance-mariadb-manual-update.xml]: Missing a full stop at the end here :)

File: xml/soc-operations-maintenance.xml [hunk 3]
```diff
@@ -1417,4 +1417,42 @@
     The &mariadb_cluster; is now running and &pacemaker; is handling the cluster.
   </para>
  </section>
+ <section>
+  <title>Updating &mariadb; with Galera</title>
+  <para>
+   When using &pacemaker;, updating &mariadb; with Galera must be done
+   manually. &crow; will not install updates automatically. In particular, this
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: &crow; does not install updates automatically.

File: xml/soc-operations-maintenance.xml [hunk 4]
```diff
@@ -1417,4 +1417,42 @@
...
+   Update &mariadb; with the following procedure:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Using the &pacemaker; GUI, put the cluster into maintenance mode. Detailed
+     information about the &pacemaker; GUI and its operation is available in the
+     <link
+     xlink:href="https://www.suse.com/documentation/sle_ha/singlehtml/book_sleha/book_sleha.html#cha.ha.configuration.gui">&sle;
+     High Availability documentation</link>
```
Comment (asettle) [inline xml/soc-operations-maintenance.xml]: missing the full stop :)

Comment (csymons-suse) [reply xml/soc-operations-maintenance.xml]: issues addressed

File: xml/operations-maintenance-mariadb-manual-update.xml (hunk 1 above)
Comment (asettle) [inline xml/operations-maintenance-mariadb-manual-update.xml]:
``` suggestion
manually. &crow; does not install updates automatically. In particular, this
```

**Pull request merged**